### PR TITLE
[Pre-Byzantium Release] v2.2.2

### PR DESCRIPTION
**Pull request by holgerd77**
Pre-byzantium release with tightened dependencies for keeping Homestead version of library installable for some time.

### CODE REVIEW

**Comment by holgerd77** [comment]
As a side effect this PR should solve the currently broken ``master`` build (internal dependency problems in installed ``standard`` version).

**Comment by holgerd77** [comment]
Please ignore the CircleCI tests, there is something wrong with the Circle setup and things are also breaking on other libraries.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ## [2.3.0] - Unreleased
 - TODO
 
+## [2.2.2] - 2017-09-19
+- Fixed issues with [handling large numbers](https://github.com/ethereumjs/ethereumjs-vm/issues/167)
```
Comment (jwasinger) [inline CHANGELOG.md]: This PR is not ready.

Comment (holgerd77) [reply CHANGELOG.md]: Updated.

**Comment by holgerd77** [comment]
Wait for ``fix-blockhash`` PR (not yet created) on ``ethereumjs-blockchain`` before merging/approving here. cc @jwasinger

**Pull request merged**